packages/serverless/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@sentry/serverless",
+  "version": "1.0.1",
```

Comment:
We release all packages in the repo at the same version number -- this needs to be updated to match all other packages.

Reply:
Version has been changed to '5.20.1'